source/settings/core.rst

```diff
@@ -1376,6 +1376,34 @@ See :ref:`settings` for list of all setting groups.
    name.
 
 
+.. dovecot_core:setting:: language
+   :seealso: @textcat_config_path;dovecot_core
+   :values: @named_filter
+   :filter: language
+
+   Defines a language to be used in tokenization.
+
+   At least one language must be specified.
+
+   The language listed first is the default and is used when language
+   recognition fails.
```

Comment:
Hmm. This is actually a bit bad. The order of named list filters isn't very clear in all situations, and I was hoping to eventually not give any guarantees about that. Maybe we need to have language_default=yes|no setting, similar to the namespace_inbox=yes|no setting that namespaces have.